linux_docker_resources/entry_point.sh

```diff
@@ -32,7 +32,7 @@ case "${CI_ARGS}" in
         ;;
       *)
         echo "Installing Connext binaries off RTI website..."
-        python3 -u /tmp/rti_web_binaries_install_script.py /tmp/rti-installer.run /home/rosbuild
+        python3 -u /tmp/rti_web_binaries_install_script.py /tmp/rti_connext_dds-5.3.0-eval-x64Linux3gcc5.4.0.run /home/rosbuild
```

Comment:
IMO it'd be better to rename the installer at the download step so the version doesn't need to be referred to in two files

Reply:
> IMO it'd be better to rename the installer at the download step so the version doesn't need to be referred to in two files

The download is not a single file anymore but an archive containing multiple files. So I would rather not touch the files from within the archive.

Reply:
I understand it's an archive but I don't see how it affects what I'm saying, that now we have to update the name of the connext web installation in multiple files. I haven't understood the reason against renaming the installer after extracting the archive?

Reply:
There are currently many locations in which the exact Connext version is hard coded. I guess I just don't see it as that valuable to reduce one of the many.

Please feel free to modify the patch in place and trigger a new CI build for it.

Reply:
no worries, was checking if I was missing something about the reasoning